azext_iot/central/command_map.py

```diff
@@ -47,13 +47,50 @@
     operations_tmpl="azext_iot.central.commands_api_token#{}"
 )
 
+central_query_ops = CliCommandType(
+    operations_tmpl="azext_iot.central.commands_query#{}"
+)
+
+central_destination_ops = CliCommandType(
+    operations_tmpl="azext_iot.central.commands_destination#{}"
+)
+
+central_export_ops = CliCommandType(
+    operations_tmpl="azext_iot.central.commands_export#{}"
+)
+
 
 # Dev note - think of this as the "router" and all self.command_group as the controllers
 def load_central_commands(self, _):
     """
     Load CLI commands
     """
 
+    with self.command_group(
+        "iot central", command_type=central_query_ops, is_preview=True
```

Comment:
The `is_preview=True` here being applied to the entire IoT Central command group which was not in preview in at least prior release. Is this meant to be applied to the query command instead?

Reply:
Yes, it should only apply to query command, any suggestions to do that?

Reply:
You can do this `cmd_group.command("query", "query_run", is_preview=True)`

Reply:
Thanks, fixed as suggested.